exp/runtime/internal/controllers/extensionconfig_controller_test.go

```diff
@@ -261,12 +261,12 @@ func TestExtensionReconciler_discoverExtensionConfig(t *testing.T) {
 
 		// Expect exactly one handler and expect the name to be the handler name plus the extension name.
 		handlers := discoveredExtensionConfig.Status.Handlers
-		g.Expect(len(handlers)).To(Equal(0))
+		g.Expect(handlers).To(HaveLen(0))
```

Comment:
should we use `BeEmpty()` here?

Reply:
@elmiko Great find! Neither I nor the linter picked up on this. But I think you are absolutely right and it's fixed everywhere.